airflow/decorators/__init__.py

```diff
@@ -15,15 +15,17 @@
 # specific language governing permissions and limitations
 # under the License.
 
-from typing import Callable, Dict, Iterable, List, Optional, Union
+from typing import Callable, Optional
+
+import importlib_metadata as metadata
 
-from airflow.decorators.python import python_task
-from airflow.decorators.python_virtualenv import _virtualenv_task
-from airflow.decorators.task_group import task_group  # noqa
```

Comment:
This one is not accessibly by `@task.task_group` -- I think you were over-zealous in your removal of stuff :D

Reply:
What do you mean by this? I'm not sure what you mean by not accessible?

Reply:
This import is just so you can do `from airflow.decorators import task_group` -- it is not used anywhere else, and `@task.task_group` isn't a thing.